frontend/src/App.css

```diff
@@ -58,7 +19,7 @@ body {
 
 /* VERTICALLYTG CENTRE */
 .isitweeka {
-  height: 100vh;
+  min-height: 100vh;
```

Comment:
```suggestion
  min-height: 100vh;
  height: 100vh;
```

Reply:
See above comment on why it goes weird without it.

Reply:
BTW not a full review, just a comment.

Reply:
looks good